rpm/lipstick-qt5.spec

```diff
@@ -47,7 +47,7 @@ BuildRequires:  qt5-qtgui-devel >= 5.2.1+git24
 BuildRequires:  qt5-qtwayland-wayland_egl-devel >= 5.4.0+git26
 BuildRequires:  doxygen
 BuildRequires:  qt5-qttools-qthelp-devel
-BuildRequires:  nemo-qml-plugin-systemsettings >= 0.5.73
+BuildRequires:  nemo-qml-plugin-systemsettings >= 0.8.0
```

Comment:
Started to wonder: Shouldn't this be Requires rather than BuildRequires?

Reply:
Requires is added automatically (albeit without exact version), and the older libraries should still work fine installed. Only build requires this for what i can tell.